packages/circuits/tests/select-regex-reveal.test.ts

```diff
@@ -77,10 +79,12 @@ describe("Select Regex Reveal", () => {
         } catch (error) {
             expect((error as Error).message).toMatch("Assert Failed");
         }
+
+        expect.assertions(1);
```

Comment:
@SoraSuegami Just FYI, we need to add this in negative testing to let jest know we expect an assertion - this ensures the catch block was executed (as our assertion is in catch). Otherwise the test would pass even if the code didn't fail as expected.